### ISSUE TITLE
App segfaults on Mac when windows get closed

### ISSUE THREAD

**Issue by ganthern** [tutao/tutanota#1004]
https://github.com/electron/electron/issues/16096
https://github.com/electron/electron/issues/15725

**Comment by ganthern** [comment]
```
Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000600020000000
Exception Note:        EXC_CORPSE_NOTIFY
```

**Comment by ganthern** [comment]
will be resolved with https://github.com/electron/electron/issues/16202

**Comment by ganthern** [comment]
will be resolved with electron 4.0.2 https://github.com/electron/electron/issues/16202
